src/agent/coverage/src/cobertura.rs

```diff
@@ -154,9 +192,9 @@ mod tests {
 
         _emitter_test.write(
             XmlEvent::start_element("class")
-                .attr("name", "0")
+                .attr("name", "C:/Users/file1.txt")
```

Comment:
This path is a funny mix of Windows and POSIX-style: it has a letter drive for the root, but uses forward slashes!

It makes me wonder: does the XML library we're using handle string escaping? What happens if our inputs are classic Windows paths like `C:\Users\file.txt`?

Reply:
It does handle, string escaping, i.e. C:\\Users

Reply:
Here's the core question: if the path is a real Windows path, with backslashes, is the resulting XML valid? How about other XML special characters (`<`, `>`, `&`, `'`, `"`)?

If not, do we need to manually invoke an XML escaping function?